### PR TITLE
Add Export to Terra tutorial (closes #512)

### LINKED ISSUE #512: Tutorial on how to send search results to Terra
There is a tutorial (on or pointed to from our “Guides” portal section) that a researcher can follow that will show them how to search and send results (could go with BAM files) to Terra. The tutorial will then show them how to do some basic analysis on the BAMs in Terra via a WDL workflow (could be the same super simple analysis that the first tutorial did).

### PR DESCRIPTION
**Pull request by natanlao**
Rich text preview [here](https://github.com/HumanCellAtlas/data-portal/blob/natanlao-terra-export/content/guides/userguides/terra-export.md).

### CODE REVIEW

File: content/guides/userguides/terra-export.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+![Terra page showing exported data](_images/terra-exported_data.png)
+
+Next, let's find a workflow to run with the data we've just exported. For this
+tutorial, we're looking for *dockstore-wdl-workflow-md5sum*, which will
+generate an MD5 checksum for a file (or files) that we provide. We'll need to
+import it from Dockstore. To do that, click on the *Workflows* tab at the top
+of the page, then on the big square *Find a Workflow* button. It'll look
+something like this:
+
+![Terra page showing workflows that can be added to workspace](_images/terra-workflows.png)
```
Comment (chmreid) [inline content/guides/userguides/terra-export.md:86]:
```suggestion
![Terra page showing workflows that can be added to workspace](_images/terra_workflows.png)
```

File: content/guides/userguides/terra-export.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+grab a coffee, stretch. You deserve it.
+
+When you come back, refresh the page. Hopefully, your workflow will be done
+running. If it is, you'll seem something like this:
+
+![Terra workflow done running](_images/terra-workflow_done.png)
+
+Note the green checkmark in the *Status* column.
+
+Congrats! If you want to see the results of this workflow executiong, click
```
Comment (chmreid) [inline content/guides/userguides/terra-export.md]:
```suggestion
Congrats! If you want to see the results of this workflow execution, click
```

File: content/guides/userguides/terra-export.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+on the workflow ID (the UUID on the right of the page), which will show the
+data generated by this workflow execution.
+
+
+## Additional documentation
```
Comment (chmreid) [inline content/guides/userguides/terra-export.md]:
```suggestion
## Additional resources
```

File: content/guides/userguides/terra-export.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+# Exporting search results from the HCA Data Explorer to Terra
+
+In this tutorial, you'll learn how to send search results from the HCA Data
+Explorer to Terra and how to run a basic workflow with that data.
+
+We'll assume that you have some familiarity with the tools aforementioned.
+In particular, you'll want to be acquainted with the content in these
+tutorials:
+
+- [Accessing HCA Data and Metadata](quick-start-guide)
```
Comment (chmreid) [inline content/guides/userguides/terra-export.md]: This link is broken

Comment (chmreid) [reply content/guides/userguides/terra-export.md]: Looks like `quick-start-guide` is missing from the links list at the bottom of this document

**Review by chmreid — CHANGES_REQUESTED** [review]
This is a clearly-written tutorial and looks great. A few details left to address though.

File: content/guides/userguides/terra-export.md (hunk 1 above)
Comment (natanlao) [reply content/guides/userguides/terra-export.md:86]: Filename in `_images` changed to match convention

**Review by chmreid — APPROVED** [review]
LGTM

File: content/guides/userguides/terra-export.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+In this tutorial, you'll learn how to send search results from the HCA Data
+Explorer to Terra and how to run a basic workflow with that data.
+
+We'll assume that you have some familiarity with the tools aforementioned.
+In particular, you'll want to be acquainted with the content in these
+tutorials:
+
+-   [Accessing HCA Data and Metadata][quick-start-guide]
+
+You'll want to use Google Chrome for this tutorial; Terra has poorer support
```
Comment (NoopDog) [inline content/guides/userguides/terra-export.md]: How is this expressed on the terra docs? Can we say terra recommends you use google chrome if that is actually the case?

File: content/guides/userguides/terra-export.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+![The *Export Selected Data* button](_images/terra-export_button.png)
+
+Click on the *Export to Terra* button. You'll then see a page like this where
+you can select what kind of data to export:
+
+![Page for choosing data to export](_images/terra-choose_files.png)
+
+Again, choose whatever looks interesting.
+
+When you click the *Request Export* button, the Data Explorer will process your
```
Comment (NoopDog) [inline content/guides/userguides/terra-export.md:68]: I think we should have some information on how to get a terra login even if it's just a line or two and a link to terra docs.

File: content/guides/userguides/terra-export.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+subTitle: ""
+---
+
+Exporting search results from the HCA Data Explorer to Terra
+============================================================
+
+In this tutorial, you'll learn how to send search results from the HCA Data
+Explorer to Terra and how to run a basic workflow with that data.
+
+We'll assume that you have some familiarity with the tools aforementioned.
```
Comment (NoopDog) [inline content/guides/userguides/terra-export.md]: Many / most  users will not be familiar with Terra as it is a new tool. I think we might add see a brief overview of what terra is and what dockstore is for folks that do not know with links out to terra and dockstore to learn more.

**Review by NoopDog — CHANGES_REQUESTED** [review]

**Comment by chmreid** [comment]
@theathorn @NoopDog since @natanlao will be out for the next week, I'll pick up this PR

File: content/guides/userguides/terra-export.md (hunk 5 above)
Comment (chmreid) [reply content/guides/userguides/terra-export.md]: From their "How to register for a Terra account" howto (https://support.terra.bio/hc/en-us/articles/360028235911-How-to-register-for-a-Terra-account):

> NOTE: Terra is designed and tested (and will run best on) on the Chrome browser. If you are having problems and are using a different browser, try switching to Chrome and see if that resolves the issue.

Comment (chmreid) [reply content/guides/userguides/terra-export.md]: Fixed working around recommended browser and added the link above

File: content/guides/userguides/terra-export.md (hunk 7 above)
Comment (chmreid) [reply content/guides/userguides/terra-export.md]: Added two sections (Overview of Terra and Overview of Dockstore) that link to relevant pages/documentation

File: content/guides/userguides/terra-export.md (hunk 6 above)
Comment (chmreid) [reply content/guides/userguides/terra-export.md:68]: Added to the "Overview of Terra" section

**Comment by chmreid** [comment]
@NoopDog Requested changes have been made, I re-requested your review on this PR. Thanks!

**Pull request merged**